alvr/xtask/src/packaging.rs

```diff
@@ -213,16 +147,16 @@ pub fn replace_client_openxr_manifest(from_pattern: &str, to: &str) {
     fs::write(manifest_path, manifest_string).unwrap();
 }
 
-pub fn package_client_openxr(flavor: ReleaseFlavor) {
-    crate::clean();
```

Comment:
these do kinda different things tho? shouldn't we keep both?

Reply:
Same goes for all the other package commands ig... (even tho those are only really run in ci when it matters)

Reply:
> these do kinda different things tho? shouldn't we keep both?

I'm not sure what you are referring to

Reply:
doing both the clean and build dir delete

Reply:
clean() does build dir delete, deps delete and target delete. since we want to run more than one foe these commands it would be better to delete neither the rust cache nor deps. nd for the build dir we just need to delete the subdir that the command writes to

Reply:
Right, tho I'd still advise to work with a 100% clean rust cache for creating releases. You never know what kinda mess of different compiler versions and rust nightlies in between can happen. Should not be an issue in practice, but I'd just opt for the safest possible option since releases are rare and the extra bit of build time shouldn't matter in that case.

(I'm a bit paranoid because I've created insane messes when using self-compiled rust compilers in the past and clean builds were the only real way to fix it)

Reply:
Maybe, yeah. I've had miscompilation bugs that resolved after a cargo clean. But in any case the CI runners are completely clean slates. The purpose of not cleaning is because we build streamer and launcher in the same runner but it shouldn't be a problem.